packages/lit-dev-content/site/tutorials/content/svg-templates/00.md

```diff
@@ -0,0 +1,14 @@
+SVG enables artists to create scalable graphics through an XML-like syntax. It's supported in most modern browsers.
+
+But beautiful graphics can get complicated!
+
+Webcomponents are a perfect match for SVG. They can encapsulate styles,
```

Comment:
Webcomponents -> Web components

SVG and WCs are also a somewhat sore topic. People tend to really complain that you can't put a WC in an SVG.

Reply:
well no where to go but up.
changed it to 
`Web components are a perfect match for distributing SVG`